integration-tests/tests/src/hooks/import-app-before.ts

```diff
@@ -48,6 +48,7 @@ export function importAppBefore(config: AppConfig | { config: AppConfig }, sdkCo
     } else {
       const { appId, baseUrl } = await importApp(config);
       this.app = new Realm.App({ id: appId, baseUrl, ...sdkConfig });
+      this.baseUrl = baseUrl;
```

Comment:
I would probably just import this directly in the test, but if you want to store this in the context, it should probably be added to the `AppContext` type as well.